ClubManagement/ClubManagement/Activities/Base/MapAbstractActivity.cs

```diff
@@ -43,10 +44,41 @@ protected void MoveMapCamera(double lat, double lng)
             googleMap.MoveCamera(cameraUpdate);
         }
 
-		public void OnMapReady(GoogleMap googleMap)
+        public void OnMapReady(GoogleMap googleMap)
         {
-			this.googleMap = googleMap;
-			MapReady?.Invoke(googleMap, EventArgs.Empty);
+            this.googleMap = googleMap;
+            MapReady?.Invoke(googleMap, EventArgs.Empty);
+            googleMap.SetInfoWindowAdapter(this);
+        }
+
+        public View GetInfoContents(Marker marker) => null;
+
+        public View GetInfoWindow(Marker marker)
+        {
+            LinearLayout info = new LinearLayout(this)
```

Comment:
Please inflate a layout instead of create from the code

Reply:
Done